detect_test.go

```diff
@@ -94,4 +108,42 @@ func testDetect(t *testing.T, context spec.G, it spec.S) {
 			}))
 		})
 	}, spec.Sequential())
+
+	context("handles pyproject.toml parser", func() {
+		it("passes WorkingDir to the pyproject.toml parser #1", func() {
+			_, _ = detect(packit.DetectContext{
+				WorkingDir: "/hi",
+			})
+			Expect(parsePythonVersion.ParsePythonVersionCall.Receives.Path).To(Equal("/hi"))
```

Comment:
I think you can make this assertion inline as part of the tests above, which is the pattern we've adopted elsewhere in this buildpack and others.

This also means you can remove the two tests to test `WorkingDir:`, as both paths are covered - setting it to an arbitrary value is covered in this unit test, and ensuring that arbitrary value isn't hard-coded will be covered by an integration test.

Reply:
done